src/overlays/actors/ovl_En_Viewer/z_en_viewer.c

```diff
@@ -146,7 +136,11 @@ static void* ActorShadowDrawFunc[] = {
 // timer
 s16 D_80B2CFCC = 0;
 
-Vec3f D_80B2CFD0 = { 0.0f, 0.0f, 0.0f };
+Vec3f D_80B2CFD0 = {
+    0.0f,
+    0.0f,
+    0.0f,
+};
```

Comment:
you can leave off the comma on shorter stuff like vectors, but its not a big deal. 
i imagine this will move inside of the function its used in eventually anyway